--- seahub/thirdpart/wsgidav/dc/seahub_db.py	2019-12-23 03:12:22.000000000 +0100
+++ seahub/thirdpart/wsgidav/dc/seahub_db.new.py	2020-01-10 11:27:10.967308951 +0100
@@ -7,6 +7,10 @@
 from sqlalchemy.pool import Pool
 from sqlalchemy.ext.automap import automap_base
 
+from seaf_utils import SEAFILE_CENTRAL_CONF_DIR, SEAFILE_CONF_DIR
+import os
+import sys
+
 Base = automap_base()
 
 import wsgidav.util as util
@@ -26,6 +30,11 @@
         return None
 
 def create_seahub_db_engine():
+    path = SEAFILE_CENTRAL_CONF_DIR if SEAFILE_CENTRAL_CONF_DIR else SEAFILE_CONF_DIR;
+    if os.path.exists(path):
+        sys.path.insert(0, path)
+
+
     import seahub_settings
     db_infos = seahub_settings.DATABASES['default']
     #import local_settings
